### PR TITLE
Get models with Batchnorm to Verify Fx after Node Ordering

### PR DESCRIPTION
**Pull request by mostafaelhoushi**
Training models with batchnorm layers have buffers. torch.fx seems to change the names of those buffers if we change the order of nodes. Hence, here we add all buffers first in order (just like we did with args and params) to avoid that issue.

Those buffers that represent moving average and moving standard deviation, need to be persistent across training iterations, hence I think it is OK to add them first to the model, as they will always stay in memory.

Test Plan:
```
python -m unittest tests.fx_optimizer_test.FXOptimizerTest.testVGG11BNNetEvalSchedule
```

To compare the fx trace of VGG11 BN before and after node ordering: https://www.diffnow.com/report/af43k

### CODE REVIEW

**Comment by mostafaelhoushi** [comment]
Verified that CI passes as well as internal tests pass in D42301044, so will merge this PR.
